src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,47 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+
+  static died(animal) {
+    const diedAnimalIndex = this.alive.indexOf(animal);
+
+    this.alive.splice(diedAnimalIndex, 1);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
-}
+  constructor() {
+    super();
+    this.hidden = false;
+  };
+
+  hide() {
+    this.hidden = true;
```

Comment:
And how can this animal stop hiding? 🙂 Better change `hidden` to opposite in this method

Reply:
I was wrong, ignore this comment 🙃